<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EATC BANK CUSTOMER UPDATE </string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(170, 170, 127);</string>
   </property>
   <property name="text">
    <string>Enter New Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(170, 170, 127);</string>
   </property>
   <property name="text">
    <string>Enter New Telephone Number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removetheaccount">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Remove Your Account</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="newpaswordedit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="newtelephoneedit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 26pt &quot;Matura MT Script Capitals&quot;;</string>
   </property>
   <property name="text">
    <string>EACT BANK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
